src/main/java/br/com/alura/comex/relatorios/CategoriaRelatorio.java

```diff
@@ -0,0 +1,38 @@
+package br.com.alura.comex.relatorios;
+
+import br.com.alura.comex.model.Pedido;
+
+import java.util.List;
+import java.util.function.Function;
+
+public enum CategoriaRelatorio {
+    SINTETICO(pedidos -> new RelatorioProxy(new RelatorioSintetico(pedidos, relatorio -> {
+        System.out.println(relatorio);
+    }))),
```

Comment:
```suggestion
    SINTETICO(pedidos -> new RelatorioProxy(new RelatorioSintetico(pedidos,  System.out::println))),
```

Esse lambda aqui pode ser trocado por um method reference!